packages/algob/src/builtin-tasks/run.ts

```diff
@@ -84,6 +85,20 @@ export async function runMultipleScriptsOneByOne (
 }
 
 export async function runMultipleScripts (
+  runtimeEnv: AlgobRuntimeEnv,
+  scriptNames: string[],
+  onSuccessFn: (cpData: CheckpointRepo, relativeScriptPath: string) => void,
+  force: boolean,
+  logTag: string,
+  allowWrite: boolean): Promise<void> {
+  try {
+    return await _runMultipleScripts(runtimeEnv, scriptNames, onSuccessFn, force, logTag, allowWrite);
+  } catch (e) {
+    if (!checkAlgorandUnauthorized(e, runtimeEnv.network)) { throw e; }
```

Comment:
If this exception occurs in multiple places then let's handle it in `cli.ts`.

Reply:
Wrapping SDK is too much code to do. I was thinking about moving it to `cli.ts`. If you think it's a better idea then I can do it.

Reply:
I think it's not a better idea because we'll lose the context of exception.
Wrapping the SDK is a task that we may think about.

Reply:
moved to `cli.ts`